### PR TITLE
Wait for effect creation, if the effect is not available yet

### PR DESCRIPTION
**Pull request by RaananW**

### CODE REVIEW

File: packages/dev/core/src/PostProcesses/postProcess.ts [hunk 1]
```diff
@@ -832,6 +838,7 @@ export class PostProcess {
             },
             this._engine
         );
+        this.onEffectCreatedObservable.notifyObservers(this._drawWrapper.effect);
```
Comment (deltakosh) [inline packages/dev/core/src/PostProcesses/postProcess.ts:841]: need to clear the observable on dispose else all good!

Comment (RaananW) [reply packages/dev/core/src/PostProcesses/postProcess.ts:841]: Done :-)

**Review by deltakosh — APPROVED** [review]

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/15389/merge

Test environment: 
https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15389/merge/index.html

To test a playground add it to the URL, for example:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15389/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/15389/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/15389/merge
https://gui.babylonjs.com/?snapshot=refs/pull/15389/merge
https://nme.babylonjs.com/?snapshot=refs/pull/15389/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/15389/merge#BCU1XR#0

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15389/merge/testResults/webgpuplaywright/index.html

**Comment by bjsplat** [comment]
WebGL2 visualization test reporter:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15389/merge/testResults/webgl2playwright/index.html

**Pull request merged**